content/apps/create-app-in-php.md

```diff
@@ -0,0 +1,129 @@
+# Create an App in PHP
+
+In this tutorial, we will provide a guide, with examples in PHP, on how to implement the parts of your App 
+required for the activation process based on OAuth 2.0 with Authorization Code.
+At the end of this tutorial, your App will receive an Access Token and will be able to call the REST API of a PIM.
+
+::: warning
+Examples in this tutorial are using PHP without any framework or library, and are consequently not following
+all the recommended best practices. We are **strongly** encouraging you to adapt those examples with the framework or 
+library of your choice.
+:::
+
+## Prerequisites
+
+You must have valid [OAuth 2.0 client credentials](/apps/using-oauth2.html#credentials).
+
+## Activation URL
+
+First, your application must expose an activation URL.  
+
+In our example, we won't do additional steps (like authentification), so we will launch the Authorization Request
+immediately in this Activation URL.
+
+Let's create an `activate.php` file:
+```php
+
+const OAUTH_CLIENT_ID = '<CLIENT_ID>';
+const OAUTH_SCOPES = '<SCOPES>'; // eg: read_products
```

Comment:
It's inside the `code`, so it will not be converted to a link.
Also, I expect them to have read the page before where we explain when to includes the scopes, or to check by themselves in the next page the scopes.

Reply:
I've removed the comment on the scopes.
We can iterate on it and add one later if needed.